src/SeleniumLibrary/keywords/browsermanagement.py

```diff
@@ -117,6 +117,70 @@ def open_browser(self, url, browser='firefox', alias=None,
         uses. Notice that prior to SeleniumLibrary 3.0, the library
         contained its own profile that was used by default.
 
+        Optional ``options`` argument allows to define browser specific
+        Selenium options. Example for Chrome, the ``options`` argument
+        allows defining the following
+        [https://seleniumhq.github.io/selenium/docs/api/py/webdriver_chrome/selenium.webdriver.chrome.options.html#selenium.webdriver.chrome.options.Options|methods and attributes]
+        and for Firefox these
+        [https://seleniumhq.github.io/selenium/docs/api/py/webdriver_firefox/selenium.webdriver.firefox.options.html?highlight=firefox#selenium.webdriver.firefox.options.Options|methods and attributes]
+        are available. Please note that not all browsers supported by the
+        SeleniumLibrary have Selenium options available, please consult
+        the Selenium documentation which browser do support the Selenium
+        options. Selenium options are also supported, when ``remote_url``
+        argument is used.
+
+        The SeleniumLibrary ``options`` argument accepts Selenium
+        options in three different formats 1) As a sting, 2) List containing
+        dictionaries 3) Python object which is an instance of the
+        Selenium options object.
+
+        The sting format is allows to define Selenium options methods
+        or attributes and it's arguments in string format. Methods and
+        attributes are separated by a comma and arguments are separated
+        with a colon. Example like this:
+        `argument:arg1,other_argument:arg2:arg3,attribute:value`.
+
+        The method and attributes are case and space sensitive
+        and must match to the Selenium options methods and attributes names.
+        It is possible to have space before and after the comma for readability.
+        Example `argument:arg1,argument:arg2` and
+        `argument:arg1 , argument:arg2` are converted to same
+        object internally. Spaces around arguments are not removed.
+        Arguments are always stings and are not converted to other types.
+        If there is need to define a literal colon for an argument,
+        it can be escaped by using literal backslash. Please
+        note that backslash is an escape character in Robot Framework and
+        therefore backslash must escaped. Example:
+        `argument:arg1_with\\\\:colon`.
```

Comment:
Is there really a need for escaping colon? I doubt attribute names can actually contain colons, so it ought to be fine to consider values like `foo:bar:zap` to mean that `foo` is the attribute and `bar:zap` is the value.

There's an actual problem escaping commas, though. Or does the code split first from colon and only then from comma? That would be somewhat weird but I haven't yet looked the code.

Reply:
Yes escaping can be needed, example for defining a proxy could have a colon. `--proxy:ip\\:port` is possible.

Reply:
I don't understand. Why wouldn't `name, value = item.split(':', 1)` work? It wouldn't handle cases like `name:with:colon:this-is-the-value` where name is `name:with:colon` and value is `this-is-the-value`, but this is only a problem if names can have colons. Method or attribute names certainly cannot.

Reply:
Some Selenium options methods take more than one argument in. Example
`method:arg, other_method:foo:bar` should be translated to 
```
options = webdriver.ChromeOptions()
options.method("arg")
options.other_method("foo", "bar")
```
But in 
`method:arg\:param`
it should be 
```
options = webdriver.ChromeOptions()
options.method("arg:param")
```
And in that proxy scenario argument contains a colon and it is the only case, which I have found, where colon is needed.